README.md

```diff
@@ -1 +1,11 @@
-# nb2
\ No newline at end of file
+# nb2
+
+An opinionated framework for network function development. Written in Rust, inspired by [NetBricks](http://netbricks.io/) and built on Intel's [Data Plane Development Kit](https://www.dpdk.org/).
+
+# Overview
+
+The project is currently undergoing heavy development. The goal is to offer an ergonomic framework for network function development that traditionally has high barriers of entry for engineers. We want a tool to efficiently manipulate network packets that's type safe, memory safe and thread safe.
```

Comment:
We are building a tool to efficiently manipulate network packets while being type-safe, memory-safe, and thread-safe.